plugin.xml

```diff
@@ -32,5 +32,16 @@
             <runs />
         </js-module>
     </platform>
+
+    <platform name="ios">
+		<source-file src="src/ios/FileChooser.swift" />
+		<dependency id="cordova-plugin-add-swift-support" version="*" />
```

Comment:
We probably should. I'll pin to 2.0.2, which is the current version, last updated 8 months ago. So it can't have changed while I was working on this :)